plugins/action_modify_record/src/modifyRecord.ts

```diff
@@ -15,6 +23,8 @@ const modifyRecord: ActionPluginType = async ({ parameters, applicationData, DBC
     ...record
   } = parameters
 
+  const tableNameProper = getValidTableName(tableName)
```

Comment:
We now strictly enforce table names. This also means we can give tables "pretty" names and use them if we like. So we can call our Products table "Registered Products" for example, and the created table with be called "data_table_registered_products", but we can keep using "Registered Products" happily in modifyRecord action, etc.

Reply:
> happily in modifyRecord action

is the table name unique?

Reply:
> is the table name unique?

What do you mean? It behaves as it currently does -- if the table name exists it adds a new record to it, if the table doesn't exist it starts a new one.